src/Compilers/Core/AnalyzerDriver/AnalyzerExecutor.cs

```diff
@@ -495,8 +495,9 @@ internal static bool IsCanceled(Exception ex, CancellationToken cancellationToke
         internal static Diagnostic GetAnalyzerExceptionDiagnostic(DiagnosticAnalyzer analyzer, Exception e)
         {
             var descriptor = new DiagnosticDescriptor(AnalyzerExceptionDiagnosticId,
-                AnalyzerDriverResources.AnalyzerFailure,
-                AnalyzerDriverResources.AnalyzerThrows,
+                title: AnalyzerDriverResources.AnalyzerFailure,
+                messageFormat: AnalyzerDriverResources.AnalyzerThrows,
+                description: string.Format(AnalyzerDriverResources.AnalyzerThrowsDescription, e.ToString()),
```

Comment:
Yes including name of the analyzer in the primary message would also be useful so that end-user can easily figure out which analyzer s/he needs to disable in the ruleset.